client/src/pages/collections/Collections.tsx

```diff
@@ -198,7 +198,17 @@ const Collections = () => {
         }
         : v
     );
-    await CollectionHttp.createCollection(data);
+
+    await CollectionHttp.createCollection({
+      ...data,
+      consistency_level: [
+        'Strong',
+        'Session',
+        'Bounded',
+        'Eventually',
+        'Customized',
+      ][data.consistency_level],
```

Comment:
we need use data.consistency_level directly instead of [xxx][xx].

Also [xxx][xx] this code style is not recommend.